### PR TITLE
99959:OTP attempt exceeded and OTP expired changes

### PR DESCRIPTION
**Pull request by ravisaurav-tarento**

### CODE REVIEW

**Comment by ravisaurav-tarento** [comment]
Hi @karthik-tarento Please review the Pull Request

File: service/src/main/java/org/sunbird/actor/otp/OTPActor.java [hunk 1]
```diff
@@ -174,9 +184,7 @@ private void handleMismatchOtp(
             + ",remaining attempt is "
             + remainingCount);
     int attemptedCount = (int) otpDetails.get(JsonKey.ATTEMPTED_COUNT);
-    if (remainingCount <= 0) {
-      otpService.deleteOtp(type, key, context);
-    } else {
+    if (remainingCount >= 0) {
```
Comment (karthik-tarento) [inline service/src/main/java/org/sunbird/actor/otp/OTPActor.java]: It should be > 0 
equal to should not come

File: core/platform-common/src/main/java/org/sunbird/exception/ResponseMessage.java [hunk 2]
```diff
@@ -110,6 +110,8 @@ interface Message {
     String SERVER_ERROR = "server error";
     String EXTENDED_USER_PROFILE_NOT_LOADED = "Failed to load extendedProfileSchemaConfig from System_Settings table";
     String ROLE_PROCESSING_INVALID_ORG = "Error while processing assign role. Invalid Organisation Id";
+    String ERROR_OTP_EXPIRED = "OTP Expired, Please Try Again.";
+    String ERROR_OTP_ATTEMPT_EXCEEDED = "Attempt Exceeded For the OTP, Please Try Again.";;
```
Comment (karthik-tarento) [inline core/platform-common/src/main/java/org/sunbird/exception/ResponseMessage.java]: Number of attempts exceeded for OTP verification. Please try again.

File: core/platform-common/src/main/java/org/sunbird/exception/ResponseMessage.java [hunk 3]
```diff
@@ -110,6 +110,8 @@ interface Message {
     String SERVER_ERROR = "server error";
     String EXTENDED_USER_PROFILE_NOT_LOADED = "Failed to load extendedProfileSchemaConfig from System_Settings table";
     String ROLE_PROCESSING_INVALID_ORG = "Error while processing assign role. Invalid Organisation Id";
+    String ERROR_OTP_EXPIRED = "OTP Expired, Please Try Again.";
```
Comment (karthik-tarento) [inline core/platform-common/src/main/java/org/sunbird/exception/ResponseMessage.java]: OTP expired, Please try again.

**Review by karthik-tarento — CHANGES_REQUESTED** [review]

File: service/src/main/java/org/sunbird/actor/otp/OTPActor.java (hunk 1 above)
Comment (ravisaurav-tarento) [reply service/src/main/java/org/sunbird/actor/otp/OTPActor.java]: I have removed equality check for this variable.

File: core/platform-common/src/main/java/org/sunbird/exception/ResponseMessage.java (hunk 3 above)
Comment (ravisaurav-tarento) [reply core/platform-common/src/main/java/org/sunbird/exception/ResponseMessage.java]: I have updated both the error messages.

**Pull request merged**